config/op-consumer/public/delta-sync-dispatching.js

```diff
@@ -64,31 +65,40 @@ async function dispatch(lib, data) {
     if (originalDeletes.length) {
       // Map deletes from OP to DL model
       const transformedDeletes = await transformStatements(fetch, deletesWithContext);
+      const filteredDeletes = removeContextTriples(transformedDeletes, originalDeletes, deletesWithContext);
```

Comment:
maybe we want to include this in the consumer itself; not in the custom code mounted per app?
Something that would return `const object = {tranformedContext, transformedStatements}`